packages/mm-backend/src/common/enums/pairs.enum.ts

```diff
@@ -0,0 +1,33 @@
+export enum PairsEnum {
```

Comment:
What's that

Reply:
It's excluded to another branch: Fork-issue_17-snapshot-processing